docs/durable_execution/prefect.md

```diff
@@ -279,6 +282,8 @@ if __name__ == '__main__':
     )
 ```
 
+(1) Each flow run executes in an isolated process, and all inputs and dependencies must be serializable. Because Agent instances cannot be serialized, instantiate the agent inside the flow rather than at the module level.
```

Comment:
```suggestion
1. Each flow run executes in an isolated process, and all inputs and dependencies must be serializable. Because Agent instances cannot be serialized, instantiate the agent inside the flow rather than at the module level.
```